package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "A JavaScript project for the Leaderboard list app",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
```

- [ ] I suggest you setup your scripts as follow:
```suggestion
   "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
```